ViewGenerator/build/ViewGenerator.targets

```diff
@@ -83,15 +83,16 @@
       <WP_Mode>development</WP_Mode>
     </PropertyGroup>
     
-    <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
```

Comment:
Probably we should check if the project optimize flag is on (that's the most important difference between release and debug... not the name)

Reply:
Yeah, i'll update it

Reply:
Apparently we don't have an 'Optimize' property in SDK format.. I'll keep it like this if I don't find a property that suits our needs